runtime/src/comm/ofi/comm-ofi.c

```diff
@@ -178,6 +178,7 @@ static chpl_bool envInjectRMA;          // env: inject RMA messages
 static chpl_bool envInjectAMO;          // env: inject AMO messages
 static chpl_bool envInjectAM;           // env: inject AM messages
 static chpl_bool envUseDedicatedAmhCores;  // env: use dedicated AM cores
+static const char *envExpectedProvider; // env: provider we should select
```

Comment:
nit: the `*` should be attached to `char` to match the other global pointers

Reply:
Ok, but I hate, hate, hate that style. Like fingernails on a chalkboard.